README.md

```diff
@@ -1,16 +1,29 @@
-# Title
+# Introduction to Active Record Associations
 
-## Learning Goals
+While you might be in the early days of writing code and learning how to
```

Comment:
I think this paragraph can be removed since it is just from the template, but otherwise, this looks good